### ISSUE TITLE
Specifying condition branching direction

### ISSUE THREAD

**Issue by R-N** [adrai/flowchart.js#151]
It doesn't work. Yes is always bottom, and no is always right. Or am I just doing it wrong?

cond(yes, right)->op1
cond(no, left) ->op2

it works for non condition, for example:

op1(right)->op3

**Comment by adrai** [comment]
do you have an example?
fyi: not every direction is possible right now...

**Comment by R-N** [comment]
Ah, okay. I actually wanted to have a branch on the left, but that's not possible yet huh

Regarding the issue, I figured that it failed when I set directions for both yes and no (even though I only swapped them). Setting just one of them works.

For the example, well, just edit the first example on the site's main page, add "right" direction for "yes" and add "bottom" direction for "no".

Or 

st=>start: Start
eb=>end: Bottom
er=>end: Right
cond=>condition: ?

st->cond
cond(yes, right)->er
cond(no, bottom)->eb

**Comment by adrai** [comment]
change `cond(no, bottom)->eb` to `cond(no)->eb`

**Comment by R-N** [comment]
It works now. Nice.

Thanks :)

**Comment by R-N** [comment]
By the way I just noticed from the commit that I can set "yes" direction to "top".
However setting "no" direction to anything other than "bottom" or "right" makes "yes" go right and "no" go bottom.

I was wondering, why set the directions for both "yes" and "no" by code by only taking the "yes" direction into account when user specify directions for both?

I tried setting "yes" direction to "top", it "broke".
So I guess it was to restrict the direction because some are still unsupported yet? But that restriction itself looks like a "bug" unless you mention it somewhere.
